subprojects/packagefiles/protobuf/meson.build

```diff
@@ -1,7 +1,15 @@
-project('protobuf', 'cpp', version: '3.12.2', license: 'protobuf')
+project('protobuf', 'cpp', version: '3.20.0', license: 'protobuf')
 
 cc = meson.get_compiler('cpp')
 
+if get_option('default_library') != 'static'
+  add_project_arguments('-DPROTOBUF_USE_DLLS', language : 'cpp')
+  if cc.get_id() == 'msvc'
+    add_project_arguments('LIBPROTOBUF_EXPORTS', language : 'cpp')
+    add_project_arguments('LIBPROTOC_EXPORTS', language : 'cpp')
```

Comment:
These are missing `-D`, and also you want to define them everywhere since they are used by GCC too.